docs/codeql/codeql-language-guides/codeql-library-for-cpp.rst

```diff
@@ -25,507 +25,731 @@ The most commonly used standard library classes are listed below.  The listing i
-| ``template <typename`` *T* ``>`` *T* *t* ``;``                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | `TemplateVariable <https://help.semmle.com/qldoc/cpp/semmle/code/cpp/Variable.qll/type.Variable$TemplateVariable.html>`__                                             | Since C++14                                                                                                                                                                                        |
-+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+This table lists Declaration_ classes representing C/C++ declarations.
 
++-------------------------------------------------------------------------------+--------------------------------------+-------------------------------------+
+|                                Example syntax                                 |             CodeQL class             |               Remarks               |
++===============================================================================+======================================+=====================================+
```

Comment:
The original version of this table unfortunately has some indentation in the first column. See https://help.semmle.com/QL/learn-ql/cpp/introduce-libraries-cpp.html#declaration-classes row 7 for the first example, but there are lots more.

I went back and rechecked the other similar topics but they didn't use this layout.

If this is difficult to fix, maybe we could ask the C/C++ team to revise the topic inline with the others?

Reply:
I meant to say - there are also formatting issues like this in other tables in this topic.